build.gradle

```diff
@@ -32,12 +33,16 @@ plugins {
 }
 
 repositories {
+    mavenLocal()
     mavenCentral()
 }
 
 java {
     sourceCompatibility = JavaVersion.VERSION_11
     targetCompatibility = JavaVersion.VERSION_11
+    java {
```

Comment:
This `java` block is redundant, it is already inside a `java` block.

Reply:
Good catch, I'll change.